Makefile.core.mk

```diff
@@ -497,6 +497,8 @@ bundle: gen helm operator-sdk ## Generate bundle manifests and metadata, then va
 				git checkout "$$csvPath" || echo "failed to revert timestamp change. assuming we're in the middle of a merge"; \
 			fi \
 		fi
+
+	@hack/patch-csv.sh bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
```

Comment:
If you do this earlier (between `helm template` and `operator-sdk generate bundle`) you can probably avoid the reformatting

Reply:
You're right! 
Even just between `helm template ... | operator-sdk generate bundle` and `operator-sdk generate kustomize manifests` (cf. [here](https://github.com/istio-ecosystem/sail-operator/blob/main/Makefile.core.mk#L486-L492)) should work